apps/web/src/components/TrendingItemSuggestion/TrendingItemSuggestion.scss

```diff
@@ -5,18 +5,18 @@ li {
   max-height: 8.75rem;
   border-radius: 0.25rem;
   padding: 1rem;
-  gap: 10;
-}
-
-.loading {
-  background-color: #d9dbdc;
+  margin-bottom: 3px;
```

Comment:
Concert to a relative unit. Whenever you find yourself typing `px` it should trigger the thought "Can I do this with a relative unit?"